src/cli/cli_commissioner.cpp

```diff
@@ -345,7 +345,11 @@ void Commissioner::HandleStateChanged(otCommissionerState aState, void *aContext
 void Commissioner::HandleStateChanged(otCommissionerState aState)
 {
     mInterpreter.mServer->OutputFormat("Commissioner: ");
+    PrintState(aState);
```

Comment:
I think `PrintState` should not print "\r\n".
```suggestion
    PrintState(aState);
    mInterpreter.mServer->OutputFormat("\r\n");
```